frontend/README.md

```diff
@@ -0,0 +1,16 @@
+# Frontend
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `yarn start`
```

Comment:
Yarn start does work because the public folder initially added after you run the `npx create-react-app` command has been removed I think.

Can you tell me if it works on your side? If so, can you commit the public folder? If not I will add it myself!